README.md

```diff
@@ -1,3 +1,27 @@
+# OMI Python Script Provider
+
+## How to build
+
+### Dependencies to build OMI Python Script Provider:
+- Install gcc and g++
+  - Redhat/CentOS: `sudo yum install gcc gcc-c++`
+  - Debian/Ubuntu OS: `sudo apt-get install gcc g++`
+- Install python and python-devel
+  - Redhat/CentOS: `sudo yum -y install python27 python27-devel`
+  - Debian/Ubuntu OS: 
+    ```
+    sudo add-apt-repository ppa:fkrull/deadsnakes

```

Comment:
I'm just checking, what is this needed for? As far as I remember I was able to install python with dev tools without this.